src/components/toolbar/Toolbar.tsx

```diff
@@ -62,6 +65,7 @@ export interface ToolbarItemProps extends ToolbarBaseProps {
   isPopover?: boolean;
   tag?: ReactNode;
   tagProps?: Omit<TagProps, 'children'>;
+  noArrow?: boolean;
```

Comment:
Have you tried to use the `minimal` prop on the `Toolbar` component? I think it does what you need.

Reply:
there is no `minimal` property over the `Toolbar` component, it is just over the ToolbarPopoverItem and the tooltip not working there all

Reply:
It is done, we remove the `noArrow ` property and property and replaced it with `minimal ` over the `Toolbar.Item` component